src/Microsoft.Diagnostics.Runtime/src/Utilities/PEImage/PEImage.cs

```diff
@@ -219,6 +219,9 @@ public void Dispose()
         /// <returns>The position in the stream of the data, -1 if the virtual address doesn't map to any location of the PE image.</returns>
         public int RvaToOffset(int virtualAddress)
         {
+            if (virtualAddress < 4096)
```

Comment:
Is 4096 the right magic number?  Does anything guarantee that sections of memory won't be laid out lower than that?  Or that the header will be always be shorter than 4096?